linalg/src/frame.rs

```diff
@@ -1,6 +1,8 @@
 #[macro_use]
 pub mod element_wise;
 
+#[macro_use]
+pub mod binary;
```

Comment:
This is actually the unicast case, right ? Should we call it bin_unicast ? (and maybe rename by_scalar bin_scalar) ?

Reply:
yep, I've done the renaming:
- binary -> unicast
- by_scalar unchanged
(just to remain short on the naming)